backpack/utils/__init__.py

```diff
@@ -1,8 +1,26 @@
 """Contains utility functions."""
+from typing import Type
 
 from pkg_resources import get_distribution, packaging
 
 TORCH_VERSION = packaging.version.parse(get_distribution("torch").version)
-VERSION_1_9_0 = packaging.version.parse("1.9.0")
-VERSION_1_8_0 = packaging.version.parse("1.8.0")
-VERSION_1_6_0 = packaging.version.parse("1.6.0")
+TORCH_VERSION_AT_LEAST_1_7_0 = TORCH_VERSION >= packaging.version.parse("1.7.0")
+TORCH_VERSION_AT_LEAST_1_8_0 = TORCH_VERSION >= packaging.version.parse("1.8.0")
+TORCH_VERSION_AT_LEAST_1_9_0 = TORCH_VERSION >= packaging.version.parse("1.9.0")
+
+
+def exception_inside_backward_pass(error: Type[Exception]) -> Type[Exception]:
+    """Returns the type of exception that gets raised inside a backward pass by PyTorch.
+
+    For Torch>=1.7.0 the error is identical.
+
+    Args:
+        error: previous exception type
+
+    Returns:
+        new exception type
+    """
+    if TORCH_VERSION_AT_LEAST_1_7_0:
+        return error
+    else:
+        return RuntimeError
```

Comment:
Note: I introduced this function to have a single interface for this kind of behaviour. It seemed like someone previously encountered the same problem. So I replaced it with this function.

Reply:
Okay.